assets/src/stories-editor/components/contexts/Snapping.js

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
I don't know if this is the right place for this component. Maybe it could get its own folder in stead of grouping it with future contexts? It might make sense to split the HoC from this component for readability.

Reply:
Don't have a strong opinion on that. Since this is the first and only context for now there's no urgency in organizing the code differently.